assets/docs/vasudevgupta7/models/wav2vec2-robust/1.md

```diff
@@ -0,0 +1,50 @@
+# Module vasudevgupta7/wav2vec2-robust/1
+
+Pre-trained speech model (without any head) from Facebook for Automatic Speech Recognition
+
+<!-- asset-path: https://storage.googleapis.com/gsoc-weights/wav2vec2_robust.tar.gz -->
+<!-- task: audio-stt -->
+<!-- network-architecture: wav2vec2 -->
+<!-- format: saved_model_2 -->
+<!-- fine-tunable: true -->
+<!-- license: apache-2.0 -->
+<!-- language: en -->
```

Comment:
The vanilla model you've published before references a Colab. Do you want to add that to these two models as well? I think it would be valuable to demonstrate how to fine-tune the model :) 
```
<!-- colab: https://colab.research.google.com/github/tensorflow/hub/blob/master/examples/colab/wav2vec2_saved_model_finetuning.ipynb -->
```

Reply:
Yeah, it would be great to add reference to that notebook. I will do that.